core-web/apps/dotcms-ui/src/app/view/components/login/dot-login-component/dot-login.component.html

```diff
@@ -74,24 +68,23 @@ <h3 class="login__title" data-testId="header">
                         loginInfo.i18nMessagesMap['password']
                     )
                 "
-                [field]="loginForm.get('password')"
-            >
+                [field]="loginForm.get('password')">
             </dot-field-validation-message>
         </div>
         <div class="login__password-settings field">
             <p-checkbox
                 [label]="loginInfo.i18nMessagesMap['remember-me']"
                 formControlName="rememberMe"
                 data-testId="rememberMe"
-                binary="true"
-            ></p-checkbox>
-            <a
+                binary="true"></p-checkbox>
+            <button
```

Comment:
This should not be a button because it change the url so it needs to be a link with text small... we need default link, if we don't have how a default link looks like, let's talk with Melissa.

Reply:
Changed!